certora/specs/SocialRecoveryModule.spec

```diff
@@ -18,10 +17,10 @@ rule recoveryModuleCanBeDisabled {
     env e;
     address prevModule;
 
-    setupRequireRecoveryModule();
+    requireSocialRecoveryModuleEnabled();
 
     safeContract.disableModule@withrevert(e, prevModule, currentContract);
     bool isReverted = lastReverted;
 
-    assert isReverted || (!isReverted && !safeContract.isModuleEnabled(currentContract));
+    assert !isReverted => !isReverted && !safeContract.isModuleEnabled(currentContract);
```

Comment:
There is a tautology here: This should be:

```
    assert !isReverted => && !safeContract.isModuleEnabled(currentContract);
```

Reply:
My bad, but I changed it to this (there is an extra `&&` in the suggested change as well.) :

```
assert !isReverted => !safeContract.isModuleEnabled(currentContract);
```

https://github.com/5afe/CandideWalletContracts/pull/19

Reply:
> there is an extra && in the suggested change as well

:see_no_evil:  - typo.